src/cdsetool/download.py

```diff
@@ -105,10 +174,20 @@ def _retry_backoff(url, session):
 
     return response
 
+def _set_proxy(options,session):
+    proxies = options.get("proxies", {})
+    if proxies != {}:
+        if "http" in proxies or "https" in proxies:
```

Comment:
This line does nothing. `requests` has the same check internally, so theres no reason for us to repeat it
```suggestion
```

Reply:
Removed.